packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx

```diff
@@ -1,42 +1,75 @@
 import React from 'react';
 import { Button, CaptionText, IconButton } from '@deriv-com/quill-ui';
-import { LabelPairedXmarkSmBoldIcon } from '@deriv/quill-icons';
+import { LabelPairedChevronsUpXlBoldIcon, LabelPairedXmarkSmBoldIcon } from '@deriv/quill-icons';
 import { Localize } from '@deriv/translations';
 import { TooltipRenderProps } from 'react-joyride';
+import { useSwipeable } from 'react-swipeable';
 
-const GuideTooltip = ({ isLastStep, primaryProps, skipProps, step, tooltipProps }: TooltipRenderProps) => (
-    <div {...tooltipProps} className='guide-tooltip__wrapper'>
-        <div>
-            {step.title && (
-                <div className='guide-tooltip__header'>
-                    <CaptionText bold color='var(--component-textIcon-inverse-prominent)'>
-                        {step.title}
+export interface GuideTooltipProps extends TooltipRenderProps {
+    setStepIndex: React.Dispatch<React.SetStateAction<number>>;
+}
+
+const GuideTooltip = ({ isLastStep, primaryProps, skipProps, step, tooltipProps, setStepIndex }: GuideTooltipProps) => {
+    const swipe_handlers = useSwipeable({
+        onSwipedUp: () => {
+            document.getElementsByClassName('trade__chart')[0].scrollIntoView();
+            setStepIndex((prev: number) => prev + 1);
+        },
+        preventDefaultTouchmoveEvent: true,
+        trackTouch: true,
+        trackMouse: true,
+    });
+
+    if (step.title === 'scroll-icon') {
+        return (
+            <div {...swipe_handlers} className='guide-tooltip__wrapper-scroll'>
+                <LabelPairedChevronsUpXlBoldIcon
+                    fill='var(--core-color-solid-slate-50)'
```

Comment:
we can also move fill inside of styles for guide-tooltip--bounce class. Up to you, it won't make big difference